src/System.Windows.Forms/src/System/Windows/Forms/ToolStripControlHost.cs

```diff
@@ -132,7 +130,7 @@ public ContentAlignment ControlAlign
         [DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden)]
         public Control Control => _control;
 
-        internal AccessibleObject ControlAccessibilityObject => Control?.AccessibilityObject;
+        internal AccessibleObject? ControlAccessibilityObject => Control?.AccessibilityObject;
```

Comment:
Why if Control can't be null here? Look at line 36

Reply:
If I remember correctly, this is related to using `null!` in `Dispose` or not.

Reply:
I think @vladimir-krestov has a point here - line:133 `public Control Control => _control;`, which means `Control` is never a null.
So depending on whether `_control.AccessibilityObject` can be null or not this should looks either:
```suggestion
        internal AccessibleObject? ControlAccessibilityObject => Control.AccessibilityObject;
```
...or
```suggestion
        internal AccessibleObject ControlAccessibilityObject => Control.AccessibilityObject;
```